### DISCUSSION TITLE
SQL execution since Dbeaver Community 24.0.3

### DISCUSSION THREAD

**Discussion by matthieukhl** [dbeaver/dbeaver#25012] [Q&A]
Hello I have been using Dbeaver Community on macOS for a few months now. I've updated to version 24.0.3 this morning. Since 24.0.3, upon executing a query with CTRL + ENTER it used to only execute the query my cursor was on. Now it executes the whole SQL script. Is this normal? Is there an option to revert back to single query execution?

Thanks :)

**Comment by E1izabeth** [comment]
Please provide the minimal script example to reproduce your issue
